### PR TITLE
assignment 3 draft 2

### PR DESCRIPTION
**Pull request by kkorn2019**
Hi Spencer, 

Not fully sure how to do the tests for the StockQuoteApplication class, so here is what I have so far. 

Since my last two assignments still aren't graded, I just copied things over to create this new branch. I will make changes here based on feedback from those. 

Thanks, 
Kayla

### CODE REVIEW

File: KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment3/StockQuoteApplication.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * This class is used to retrieve stock quotes
+ */
+public class StockQuoteApplication {
+    
+    public static void main(String[] args){
```
Comment (smarks) [inline KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment3/StockQuoteApplication.java]: don't forget to document all methods including this main method.

File: KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment3/StockQuoteApplication.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    }
+    
+        public static List<StockQuote> createListOfStockquotes(String symbol, String from, String until){
+            SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/YYYY");
+            Calendar fromDate = Calendar.getInstance();
+            
+            try{
+                fromDate.setTime(sdf.parse(from));
+            }catch(ParseException e){
+                e.printStackTrace();
```
Comment (smarks) [inline KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment3/StockQuoteApplication.java]: Never just catch an exception and print the stack trace. handle the exception instead. Please review my notes on exception handling as well as chapter in Bloch. Let me know if you have any questions.

File: KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment3/StockQuoteApplication.java (hunk 1 above)
Comment (kkorn2019) [reply KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment3/StockQuoteApplication.java]: Added javadoc throughout.

File: KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment3/StockQuoteApplication.java (hunk 2 above)
Comment (kkorn2019) [reply KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment3/StockQuoteApplication.java]: Added exception handling.

**Pull request merged**